### DISCUSSION TITLE
Directions to create a new repository should say `echo "# <repo>" > README.md`

### DISCUSSION THREAD

**Discussion by lydiastepanek** [community/community#111151] [Repositories] [answered]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

When I create a new repository in Github, I see the following directions:
![Screenshot 2024-03-06 at 4 38 37 PM](https://github.com/community/community/assets/7087489/00f46574-aa75-4c23-9f1f-d52c3b30e2be)

Instead of `echo "# too-phishy-chrome-extension" >> README.md`, shouldn't it be `echo "# too-phishy-chrome-extension" > README.md`? Since there is no README.md in my brand new repository.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by MostafaWaleed0** [comment, marked as answer]
Yes, you're correct. If you're creating a new repository on GitHub and want to initialize it with a README file using the command line, you would typically use the `>` operator instead of `>>`.

Here's the difference between the two:

`>`: Redirects the output of a command and writes it to a file. If the file already exists, it will overwrite its contents.
`>>`: Appends the output of a command to a file. If the file doesn't exist, it creates a new one. If it exists, it adds the output to the end of the file.

Since you want to initialize your new repository with just the content `# too-phishy-chrome-extension` in the `README.md` file, you should use `>` to create the file and write to it:

`
echo "# too-phishy-chrome-extension" > README.md
`

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lydiastepanek ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Jennykehe** [comment]
> Yes, you're correct. If you're creating a new repository on GitHub and want to initialize it with a README file using the command line, you would typically use the `>` operator instead of `>>`.
> 
> Here's the difference between the two:
> 
> `>`: Redirects the output of a command and writes it to a file. If the file already exists, it will overwrite its contents.
> `>>`: Appends the output of a command to a file. If the file doesn't exist, it creates a new one. If it exists, it adds the output to the end of the file.
> 
> Since you want to initialize your new repository with just the content `# too-phishy-chrome-extension` in the `README.md` file, you should use `>` to create the file and write to it:
> 
> `
> echo "# too-phishy-chrome-extension" > README.md
> `

**Comment by Jennykehe** [comment]
J
